src/reconcile/rec_visibility.c

```diff
@@ -112,6 +112,15 @@ __rec_append_orig_value(
             break;
     }
 
+    /*
+     * We end up in this function because we have selected a newer value to write to disk. For
```

Comment:
@quchenhao 
Just for my own understanding:
So the update we selected just got aborted. Does reconciliation fail after this?

Reply:
I talked to Hari. We can't resolve the prepared update if the page is in eviction. It will wait until the eviction finishes.